### PR TITLE
Update NuGet dependencies

### PR DESCRIPTION
**Pull request by AtariDreams**
Some of these dependencies were even in beta and superseded by a release version

### CODE REVIEW

**Comment by DrusTheAxe** [comment]
Great catch! Thanks we'll look into it

@kythant @Scottj1s tag you're it

File: test/AppLifecycle/AppLifecycle.vcxproj.filters [hunk 1]
```diff
@@ -50,5 +50,6 @@
       <Filter>CopyFiles</Filter>
     </CopyFileToFolders>
     <CopyFileToFolders Include="$(TestPkgOutputFile)" />
+    <CopyFileToFolders Include="$(TestPkgOutputFile)" />
```
Comment (kythant) [inline test/AppLifecycle/AppLifecycle.vcxproj.filters:53]: Seems like a duplicate here. Please take a look and fix.

**Comment by AtariDreams** [comment]
Done @kythant

**Comment by bpulliam** [comment]
After rebasing to HEAD, no changes remain. Closing PR. I again apologize for our delayed response on this and thank you for your contribution.

**Pull request closed without merging**